doc/schema.md

```diff
@@ -0,0 +1,73 @@
+
+ifm3d - Schema Description and usage
+====================================
+
+`ifm3d` provides `FrameGrabber` class to grab the data from the device,
+ifm devices provide many type of data and images, user can configure
+`FrameGrabber` for data required by the application, this reduces
+the network traffic while grabbing the data. To configure the `FrameGrabber`
+object for particular image or/and data user needs to provide the 16 bit value,
+called as the schema mask. 12 bits out of 16 bits are used to configure the 12 types
+data and images provided by ifm 3d vision devices. 4 bits are reserved for
+future. To create schema mask bitwise OR the 16 bit values of the
+data/Image needed for application
```

Comment:
I don't understand the meaning of this sentence. The 4 additional bits are used to create the full schema mask?